source/more-info/dockerhub-rate-limit.markdown

```diff
@@ -0,0 +1,36 @@
+---
+title: "Docker Hub Rate Limit"
+description: "Docker Hub are rate limiting how many pull you can do."
+---
+
+## The issue
+
+Docker Hub enforces a limit on how much you are allowed to fetch container information from their container registry [read more about how it's handled here][docker-rate-limit].
+
+When your IP address is rate limited updating our containers will fail.
+
+## The solutions
+
+If you are running watchtower or similar solutions to keep your containers up to date, you need to reconfigure those to check less often than the default configuration for these. You should also consider removing them completely if you are running a Supervised installation since running those alongside the Supervisor is [not supported][unsupported-container].
```

Comment:
```suggestion
If you are running watchtower or similar solutions to keep your containers up to date, you need to reconfigure them to check less often than the default configuration. If you are running a Supervised installation, you should also consider removing them completely since running those alongside the Supervisor is [not supported][unsupported-container].
```